src/blog/test-the-future.md

```diff
@@ -1,7 +1,9 @@
 ---
 title: 'Help us test the future of V8!'
-author: 'Daniel Clifford ([@expatdanno](https://twitter.com/expatdanno)), Original Munich V8 Brewer'
+author: 'Daniel Clifford ([@expatdanno](https://twitter.com/expatdanno)), Original Munich V8 Brewer, anlex N'
```

Comment:
The proposed changes don't qualify for adding yourself as an author. Please remove this change here.